### DISCUSSION TITLE
Direct payment to exchange walet

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#2242] [Technical Support] [answered]
Hello,

I wanted to know wether it is possible or not to get the payments done directly to an exchange wallets instead of internal wallets.
The goal of such thing would be to avoid paying a transaction fee an extra time.

For example, if I propose to get paid in bitcoin, is it possible to have these bitcoins sent directly to my binance bitcoin wallet instead of having to get paid on my internal bitcoin wallet and then transfer the funds to binance?

**Comment by Kukks** [comment, marked as answer]
This is definitely impossible. There is no way to detect payments using only 1 address for all invoices.

**Reply by ghost** [reply]
doesn't the binance api or the coinbase api give the permission to create new addresses for the wallet? Wouldn't it be possible to generate a new address on the binance wallet for each new invoice?
Maybe it is impossible now but would be possible as a new feature?

**Reply by Kukks** [reply]
This is not a simple feature, and not something we want to build in as a native feature as it encourages bad habits of storing bitcoin in custodian wallets.  Maybe as a plugin in the future.
